lms/tasks/email_digests.py

```diff
@@ -148,7 +148,7 @@ def send_instructor_email_digest(
     created_before = datetime.fromisoformat(created_before)
 
     if created_after is None:
-        created_after = created_before - timedelta(days=7)
+        created_after = created_before - timedelta(days=7)  # type: ignore
```

Comment:
Changing the type of a variable is usually frowned upon, we go from str -> datetime above.

We could do a manual `cast` but ignored these two works for now.